src/components/sbb-timetable-row/sbb-timetable-row.helper.tsx

```diff
@@ -1,32 +1,25 @@
 import { h, JSX } from '@stencil/core';
-import {
-  differenceInDays,
-  differenceInHours,
-  differenceInMinutes,
-  subHours,
-  addMinutes,
-  subDays,
-} from 'date-fns';
+import { differenceInDays, differenceInHours, differenceInMinutes, addMinutes } from 'date-fns';
 import getDocumentLang from '../../global/helpers/get-document-lang';
 import { i18nDurationMinute, i18nDurationHour } from '../../global/i18n';
 import { PTRideLeg } from '../../global/interfaces/pearl-chain-properties';
-import { Notice, PtSituation, Trip, VehicleModeEnum } from './sbb-timetable-row.custom';
+import { HimCus, Notice, PtSituation, Trip, VehicleModeEnum } from './sbb-timetable-row.custom';
 
 export const durationToTime = (duration: number): string => {
   const result = [];
-  const now = 0;
-  let future = addMinutes(now, duration);
+  const now = new Date();
```

Comment:
For me it doesn't make sense to start from the current date. Even if the output is the same, it's always a different calculation. Do I miss something? My alarm bells always ring if there is a "new Date()" in the code somewhere ;)

Reply:
I thought it was a good idea to refactor out the let variable. But i reverted the change.

Reply:
I think you can also do it with your logic, but set now to 0 anyways ;) As you prefer.